packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarModule.tests.js

```diff
@@ -197,6 +243,78 @@ const dialogModuleConfig = {
     }
 };
 
+const dialogAIModuleConfig = {
+    beforeEach: function() {
+        fx.off = true;
+
+        this.$element = $('#htmlEditor');
+        this.log = [];
+        this.focusStub = sinon.stub();
+
+        this.quillMock = {
+            format: (format, value) => {
+                this.log.push({ format: format, value: value });
+            },
+            insertText: (index, text, formats) => {
+                this.log.push({ index: index, text: text, formats: formats });
+            },
+            insertEmbed: (index, type, value) => {
+                this.log.push({ index: index, type: type, value: value });
+            },
+            on: noop,
+            off: noop,
+            keyboard: {
+                addBinding: noop
+            },
+            table: {
```

Comment:
Do we really need all this stuff? Isn't there existing moch we can use? Or at least minify it?